lib/base-container.js

```diff
@@ -39,6 +39,12 @@ export default class BaseContainer {
 		return driverHelper.waitTillPresentAndDisplayed( this.driver, this.expectedElementSelector, this.explicitWaitMS );
 	}
 
+	waitRequestsToComplete() {
+		return this.driver.wait( () => {
+			return this.driver.executeScript( 'return jQuery.active === 0' );
```

Comment:
This method should also check to make sure that jQuery is loaded also because sometimes switching iFrames there are delays (which is why we have the sleeps to begin with)